browser/static/js/main_zstack.js

```diff
@@ -1093,6 +1093,20 @@ function action(action, info, frame = current_frame) {
 }
 
 function start_caliban(filename) {
+  // disable scrolling from scrolling around on page (it should just control brightness)
```

Comment:
not just brightness right? also scales color (e.g. label scale)

Reply:
That feature (adjusting color range of labels) is not included in browser. It would require server calls to adjust and reapply the colormap to the label array, then send back a png. Between the modified viridis colormap that browser uses and highlighting, I don't think it's necessary to include. That's why scrolling only affects brightness of raw image

Reply:
👍